diff -urN libexif-0.6.13.org/libexif/libexif.pc.in libexif-0.6.13/libexif/libexif.pc.in
--- libexif-0.6.13.org/libexif/libexif.pc.in	2005-03-17 16:58:18.000000000 +0100
+++ libexif-0.6.13/libexif/libexif.pc.in	2006-03-25 11:36:12.422870250 +0100
@@ -1,7 +1,7 @@
 prefix=@prefix@
 exec_prefix=@exec_prefix@
 libdir=@libdir@
-includedir=@includedir@
+includedir=@includedir@/libexif
 
 Name: libexif
 Description: Library for easy access to EXIF data
