--- libexif-0.6.11/configure.in.orig	2004-10-16 14:20:09.000000000 +0200
+++ libexif-0.6.11/configure.in	2004-10-17 23:51:34.190366200 +0200
@@ -46,7 +46,7 @@
 else
 	AC_MSG_RESULT([no])
 fi
-AM_GNU_GETTEXT
+AM_GNU_GETTEXT([external])
 
 
 dnl ---------------------------------------------------------------------------
@@ -59,7 +59,7 @@
     AC_SUBST(LDFLAGS)
 fi
 
-AC_OUTPUT([
+AC_OUTPUT([ po/Makefile.in
   Makefile
   libexif.spec
   libexif/Makefile
